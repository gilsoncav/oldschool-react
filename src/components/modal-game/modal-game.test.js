import ModalGame from './modal-game';

describe('ModalGame component', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<ModalGame visible />);
    expect(wrapper).toMatchSnapshot();
  });
});
