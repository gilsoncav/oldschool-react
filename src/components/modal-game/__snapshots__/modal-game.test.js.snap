// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ModalGame component renders correctly 1`] = `
<Modal
  confirmLoading={false}
  footer={null}
  maskTransitionName="fade"
  okType="primary"
  onCancel={[Function]}
  onOk={[Function]}
  title="Super Mario Kart"
  transitionName="zoom"
  visible={true}
  width={520}
>
  <Button
    block={false}
    ghost={false}
    htmlType="button"
    loading={false}
    onClick={[Function]}
    style={
      Object {
        "left": "85%",
        "position": "absolute",
        "top": "15%",
      }
    }
    type="primary"
  >
    FAV
  </Button>
  <h1
    className="text-center text-primary"
  >
    Super Mario Kart
  </h1>
  <div
    className="text-center"
    style={
      Object {
        "padding": "15px",
      }
    }
  >
    <ReactPlayer
      config={
        Object {
          "dailymotion": Object {
            "params": Object {
              "api": 1,
              "endscreen-enable": false,
            },
            "preload": false,
          },
          "facebook": Object {
            "appId": "1309697205772819",
          },
          "file": Object {
            "attributes": Object {},
            "dashVersion": "2.9.2",
            "forceAudio": false,
            "forceDASH": false,
            "forceHLS": false,
            "forceVideo": false,
            "hlsOptions": Object {},
            "hlsVersion": "0.10.1",
            "tracks": Array [],
          },
          "mixcloud": Object {
            "options": Object {
              "hide_cover": 1,
            },
          },
          "soundcloud": Object {
            "options": Object {
              "buying": false,
              "download": false,
              "liking": false,
              "sharing": false,
              "show_comments": false,
              "show_playcount": false,
              "visual": true,
            },
          },
          "twitch": Object {
            "options": Object {},
          },
          "vimeo": Object {
            "playerOptions": Object {
              "autopause": false,
              "byline": false,
              "portrait": false,
              "title": false,
            },
            "preload": false,
          },
          "wistia": Object {
            "options": Object {},
          },
          "youtube": Object {
            "embedOptions": Object {},
            "playerVars": Object {
              "iv_load_policy": 3,
              "modestbranding": 1,
              "playsinline": 1,
              "rel": 0,
              "showinfo": 0,
            },
            "preload": false,
          },
        }
      }
      controls={false}
      height="inherit"
      light={false}
      loop={false}
      muted={false}
      onBuffer={[Function]}
      onBufferEnd={[Function]}
      onDisablePIP={[Function]}
      onDuration={[Function]}
      onEnablePIP={[Function]}
      onEnded={[Function]}
      onError={[Function]}
      onPause={[Function]}
      onPlay={[Function]}
      onProgress={[Function]}
      onReady={[Function]}
      onSeek={[Function]}
      onStart={[Function]}
      pip={false}
      playbackRate={1}
      playing={false}
      playsinline={false}
      progressInterval={1000}
      style={Object {}}
      url={null}
      volume={null}
      width="inherit"
      wrapper="div"
    />
  </div>
  <div
    className="text-center"
  >
    <strong>
      Platform: 
    </strong>
    <span>
      Super Nintendo Entertainment System
    </span>
  </div>
  <div
    className="text-center"
  >
    <strong>
      Overview: 
    </strong>
    <span>
      The best kart game in the world, y' now
    </span>
  </div>
</Modal>
`;
